opensfm/src/third_party/libmv/tools/tool.h

```diff
@@ -34,8 +34,8 @@ namespace libmv {
 
 inline void Init(const char *usage, int *argc, char ***argv) {
   google::InitGoogleLogging((*argv)[0]);
-  google::SetUsageMessage(std::string(usage));
-  google::ParseCommandLineFlags(argc, argv, true);
+  gflags::SetUsageMessage(std::string(usage));
+  gflags::ParseCommandLineFlags(argc, argv, true);
```

Comment:
Distribution packages on Ubuntu 14.4 and brew/Mac seem to come with the old google namespace. We can add
```c++
#ifndef gflags
namespace gflags=google
#endif
```
to make the code work on those platforms.